modules/nlu/src/backend/tools/encoder.ts

```diff
@@ -0,0 +1,19 @@
+import _ from 'lodash'
+
+export function encodeOH(classes: string[] | number[], values: string[] | number[]): number[] {
```

Comment:
encodeOH ? Can you replace it with the full name, or add a comment explaining what this method does?

Reply:
this isn't used anymore, I remove it